### PR TITLE
feat: Add OSCAL to each commit

### PR DESCRIPTION
**Pull request by JonZeolla**

### CODE REVIEW

**Comment by brandtkeller** [comment]
I like the automation - the `upload-artifact` use of the zip feels kinda silly (given the minute space savings) but looks a constraint of the action - so no big deal there.

Curious on your explicit use of `1.0.4` of OSCAL versus a new version? Are there dependencies to this version that prevent using a newer release?

**Comment by JonZeolla** [comment]
@brandtkeller it's the latest that compliance-trestle supports

https://github.com/IBM/compliance-trestle#important-note

File: .github/workflows/commit.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+jobs:
+  generate-oscal:
+    name: Generate OSCAL
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```
Comment (JonZeolla) [inline .github/workflows/commit.yml:27]:
```suggestion
```

File: .github/workflows/commit.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    shell: 'bash --noprofile --norc -Eeuo pipefail {0}'
+
+jobs:
+  generate-oscal:
+    name: Generate OSCAL
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
```
Comment (JonZeolla) [inline .github/workflows/commit.yml:26]:
```suggestion
```

**Comment by brandtkeller** [comment]
Makes sense! thanks for the context. Working with versions myself currently and may need to ping the trestle team to see what their experience is.

**Comment by JonZeolla** [comment]
@brandtkeller awesome. Yeah this is definitely MVP for now but once it shipped then we can iterate on modernizing the version of OSCAL, adding more helpful metadata, etc.

**Review by brandtkeller — APPROVED** [review]

**Pull request merged**